src/main/java/com/epam/tc/hw2/trello/asserts/TrelloAssertProvider.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.tc.hw2.trello.asserts;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+import io.restassured.filter.log.LogDetail;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import org.apache.http.HttpStatus;
+
+public class TrelloAssertProvider {
+
+    ValidatableResponse responseToCheck;
+
+    public static TrelloAssertProvider assertThat() {
+        return new TrelloAssertProvider();
+    }
+
+    public TrelloAssertProvider response(Response response) {
+        responseToCheck = response.then().log().ifValidationFails(LogDetail.BODY);
+        return this;
+    }
+
+    public TrelloAssertProvider checkCorrectJsonFormat() {
+        responseToCheck.assertThat().contentType(ContentType.JSON);
+        return this;
+    }
+
+    public TrelloAssertProvider checkCorrectId(String id) {
+        responseToCheck.body("id", is(id));
```

Comment:
checking should be on the more low level with rest assured
extact, body ect..

Reply:
Done